src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaHumanExperimentsController.java

```diff
@@ -21,7 +26,9 @@ public HcaHumanExperimentsController(HcaHumanExperimentService hcaHumanExperimen
     //Used by anatomogram experiments table in HCA Landing page
     @GetMapping(value = "/json/experiments/hca/human",
             produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
-    public String getOrganismPartExperimentsList(@RequestParam(defaultValue = "") String organismPart) {
+    public String getOrganismPartExperimentsList(@RequestParam(defaultValue = "") Set<String> organismPart) {
+        LOGGER.info("Number of organism parts: {}",organismPart.size());
```

Comment:
I guess there is no need for logging this information

Reply:
Removed.